## This code was put into the public domain by Jens Peter Secher <jps@debian.org>.

Index: libpam-ssh/pam_ssh.8
===================================================================
--- libpam-ssh.orig/pam_ssh.8	2009-03-30 23:39:23.000000000 +0200
+++ libpam-ssh/pam_ssh.8	2009-03-30 23:40:04.000000000 +0200
@@ -57,7 +57,7 @@
 decrypt at least one of the user's SSH login keys using that passphrase.
 .Pp
 The following options may be passed to the authentication module:
-.Bl -tag -width ".Cm use_first_pass"
+.Bl -tag -width ".Cm allow_blank_passphrase"
 .It Cm debug
 .Xr syslog 3
 debugging information at
@@ -87,6 +87,8 @@
 is the first module on the stack,
 or if no previous modules
 obtained the user's password.
+.It Cm allow_blank_passphrase
+Allow SSH keys with no passphrase.
 .El
 .Pp
 If neither
